src/fmeval/eval_algorithms/helper_models/helper_model.py

```diff
@@ -114,7 +119,19 @@ def __init__(self, column_name: str = COLUMN_NAME):
 
         :param column_name: column name used to fetch input texts in __call__ method
         """
-        self._model = Detoxify(model_type="unbiased").predict
+        state_dict = torch.hub.load_state_dict_from_url(self.UNBIASED_MODEL_URL, map_location="cpu")
+        config = state_dict["config"]["arch"]["args"]
+        self._model = (
+            getattr(transformers, config["model_name"])
+            .from_pretrained(
+                pretrained_model_name_or_path=None,
+                config=AutoConfig.from_pretrained(config["model_type"], num_labels=config["num_classes"]),
+                state_dict=state_dict["state_dict"],
+                local_files_only=False,
+            )
+            .to("cpu")
```

Comment:
I think this is fine for now, but we should ideally identify whether any GPUs exist, and if so, place the model on the GPU instead.